apps/native-component-list/src/screens/ActionSheetScreen.tsx

```diff
@@ -52,46 +55,46 @@ class App extends React.Component {
         />
         <ShowActionSheetButton
           title="Title"
-          withTitle
+          withTitle={true}
```

Comment:
TypeScript requires that? 😱

Reply:
Oo ... I had provided unintentionally a rule for tslint `jsx-boolean-value` ([see here](https://github.com/palantir/tslint-react#rules)). I will turn it off. I'll turn it off.